tests/test_timefuncs.py

```diff
@@ -140,6 +140,15 @@ def test_timedelta_series_arithmetic() -> None:
     r4: pd.TimedeltaIndex = tds1 / 10.2
 
 
+def test_timedelta_series_string() -> None:
+    tds1: pd.TimedeltaIndex = pd.to_timedelta(["1 day"])
+    td1: pd.Timedelta = pd.Timedelta("2 days")
+    r1: pd.TimedeltaIndex = tds1 + td1
+    r2: pd.TimedeltaIndex = tds1 - td1
+    r3: pd.TimedeltaIndex = tds1 * 4.3
+    r4: pd.TimedeltaIndex = tds1 / 10.2
```

Comment:
You don't need these tests here.

You do need to do the test as follows:
```python
check(assert_type(pd.to_timedelta(["1 day"]), pd.TimedeltaIndex), pd.TimedeltaIndex)
```

Reply:
Thanks for the feedback and the clarity!